components/persistor/test/persist_banking_s3.js

```diff
@@ -23,6 +25,10 @@ const Customer = PersistObjectTemplate.create('Customer', {
         isRemoteObject: true,
         remoteKeyBase: 'test-remote-key',
         value: null
+    },
```

Comment:
Why are we still doing tests in JS?

Reply:
good question! given that this needs to get merged by tomorrow afternoon I used the existing tests to keep the testing footprint small

Reply:
Resolved offline. Will create a ticket and prioritize